src/AppServices/KafkaPageV3.tsx

```diff
@@ -0,0 +1,335 @@
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardHeaderMain,
+  CardTitle,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Divider,
+  Flex,
+  FlexItem,
+  Grid,
+  Split,
+  SplitItem,
+  Stack,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+} from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { AppSpeedKeyArt, LogoAWS, LogoAzure, LogoGCP } from "../images";
+import {
+  KafkaInstanceCapacityTable,
+  MarketingPageHero,
+  MarketingPageSection,
+} from "./components";
+import { ExternalLink } from "../shared";
+
+export const KafkaPageV3: FunctionComponent = () => {
+  const { t } = useTranslation(["kafkaoverview-v3"]);
+  return (
+    <>
+      <MarketingPageHero
+        title={t("heroTitle")}
+        tagLine={t("heroTagline")}
+        description={t("heroDescription")}
+        heroImage={AppSpeedKeyArt}
+        heroImageSize={478}
+        heroImageCanRepeat={false}
+        heroImagePositionY={-99}
+      />
+      <MarketingPageSection>
+        <Grid hasGutter lg={6}>
+          <Card
+            ouiaId="card-overview-purchase-now"
+            aria-label={t("kafkaoverview:purchaseCardTitle")}
+          >
+            <CardHeader>
+              <CardHeaderMain>
+                <CardTitle>
+                  <Title headingLevel="h2">
+                    <Trans
+                      ns={"kafkaoverview-v3"}
+                      i18nKey={"purchaseCardTitle"}
+                    >
+                      <span className="pf-u-color-200 pf-u-font-size-sm"></span>
+                    </Trans>
+                  </Title>
+                </CardTitle>
+              </CardHeaderMain>
+            </CardHeader>
+            <CardBody>{t("purchaseCardMainText")}</CardBody>
+            <CardFooter>
+              <Flex>
+                <FlexItem>
+                  <ExternalLink
+                    href={
+                      "https://marketplace.redhat.com/en-us/products/red-hat-openshift-streams-for-apache-kafka"
+                    }
+                    testId={"cardPurchase-buttonRH"}
+                    className={"pf-u-ml-md "}
+                  >
+                    <Trans
+                      ns={"kafkaoverview-v3"}
+                      i18nKey={"purchaseCardCallToActionButtonRedHat"}
+                      components={{
+                        bold: <strong />,
+                        size: <span className="pf-u-font-size-lg" />,
+                      }}
+                    />
+                  </ExternalLink>
+                </FlexItem>
+
+                <FlexItem>
+                  <ExternalLink
```

Comment:
With the current spacing between these buttons, the icon is equally spaced between the text. Ideally, it should be closer to the text label of the link it belongs to, and further from the text in the other link.
<img width="143" alt="image" src="https://user-images.githubusercontent.com/21063328/180331399-90df318f-c65f-4678-8aa5-dfe578c8ce90.png">

There's an example of how to modify the space between flex item here: https://www.patternfly.org/v4/layouts/flex#spacing-xl

I suggest using 2xl for this one. Here's an example using 2xl spacing:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/21063328/180331699-bd054585-3849-46bb-b790-ac1eb33ce6c7.png">